packages/wallets/src/features/cashier/modules/TransactionStatus/components/TransactionStatusSuccess/TransactionStatusSuccess.tsx

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import { useHistory } from 'react-router-dom';
+import { WalletButton, WalletText } from '../../../../../../components/Base';
+import { THooks } from '../../../../../../types';
+import useRecentTransactions from '../../hooks/useRecentTransactions';
+import { CryptoTransaction } from '../CryptoTransaction';
+
+type TTransactionStatusSuccess = {
+    recentTransactions: ReturnType<typeof useRecentTransactions>['recentTransactions'];
+    transactionType?: THooks.CryptoTransactions['transaction_type'];
+    wallet: THooks.ActiveWalletAccount;
+};
+
+const TransactionStatusSuccess: React.FC<TTransactionStatusSuccess> = ({
+    recentTransactions,
+    transactionType,
+    wallet,
+}) => {
+    const history = useHistory();
+
+    const filteredTransactions =
+        recentTransactions?.filter(
+            el => !transactionType || (transactionType === 'deposit' ? el.is_deposit : el.is_withdrawal)
+        ) || [];
+
+    return (
+        <React.Fragment>
+            {filteredTransactions?.length > 0 ? (
+                <React.Fragment>
+                    {filteredTransactions?.slice(0, 3).map((transaction, index) => (
+                        <React.Fragment key={transaction.id}>
+                            <CryptoTransaction
+                                currencyDisplayCode={wallet.currency_config?.code || ''}
+                                key={transaction.id}
+                                transaction={transaction}
+                            />
+                            <div
+                                className={
+                                    index < filteredTransactions.length - 1 && index < 2
+                                        ? 'wallets-transaction-status-success__divider--light'
+                                        : 'wallets-transaction-status-success__divider'
```

Comment:
This could also be done using the `Divider` base component where the colour of the `Divider` could be passed as an optional prop with '#f2f3f4' as the default colour. That way it could be implemented for all the cases here without having to duplicate the css properties of the different dividers used on the different files

Reply:
Using `Divider` now